session.go

```diff
@@ -1,97 +1,77 @@
 package bond
 
 import (
+	"database/sql"
+	"fmt"
 	"reflect"
 	"sync"
 
-	"upper.io/db"
+	"upper.io/db.v2"
+	"upper.io/db.v2/lib"
 )
 
 type Session interface {
-	db.Tx
+	lib.SQLDatabase
+
 	Store(interface{}) Store
 	Find(...interface{}) db.Result
 	Save(Model) error
 	Delete(Model) error
-	NewTransaction() (Session, error)
-	ContinueTransaction() (Session, error, bool)
+
+	BondTx(func(tx Session) error) error
```

Comment:
got it, that makes sense. Let's rename it to SessionTx(Session) then.. this way, we have Tx() for upper conns, and SessionTx() for bond Sessions. its more clear